README.md

```diff
@@ -82,7 +81,7 @@ wget -O - https://github.com/palomachain/paloma/releases/download/v0.8.0/paloma_
   tar -C /usr/local/bin -xvzf - palomad
 ```
 
-2. Setup your pigeon by following [the instructions](https://github.com/palomachain/pigeon#install) and then come back here.
+2. Setup your pigeon relayer by following [the instructions](https://github.com/palomachain/pigeon#install) and then come back here.
```

Comment:
Extremely minor, but if you link a pertinent part of the sentence `[Set up your pigeon relayer](linky) by following the linked instructions` it's much easier for screen readers to tab through and pick it up.

Reply:
Agreed. Updated.